tests/common/fixtures/advanced_reboot.py

```diff
@@ -116,7 +117,9 @@ def __extractTestParam(self):
         if self.rebootLimit is None:
             if self.kvmTest:
                 self.rebootLimit = 200 # Default reboot limit for kvm
-            elif 'warm-reboot' in self.rebootType:
+            if self.other_vendor_nos:
+                self.rebootLimit = 50 # Threshold set for reboot from other vendor
```

Comment:
Why is this 50s expectation set? I don't think this is correct. Expectation for fastboot from vendor NOS is still <30s.

Reply:
The flushing of the DBs is increasing the downtime in a few seconds, we would like to push it with the current downtime.